app/repos/people.js

```diff
@@ -232,25 +244,29 @@ const getPersonAndDogsByReference = async (reference, transaction) => {
   }
 }
 
+const createContact = async (person, type, contact, transaction) => {
+  const contactType = await getContactType(type)
+
+  const email = await sequelize.models.contact.create({
```

Comment:
Very minor but 'email' is a confusing term - probably better to call it 'contact' ?